ec/src/models/mnt6/mod.rs

```diff
@@ -25,7 +25,7 @@ pub type GT<P> = Fp6<P>;
 pub trait MNT6Parameters: 'static {
     const TWIST: Fp3<Self::Fp3Config>;
     const TWIST_COEFF_A: Fp3<Self::Fp3Config>;
-    const ATE_LOOP_COUNT: &'static [u64];
+    const ATE_LOOP_COUNT_2: &'static [i8];
```

Comment:
Why is this called `ATE_LOOP_COUNT_2`? I think we can just leave it as the previous one.

Reply:
Yes I guess so. The "2" in the name was partly inspired by [bw6](https://github.com/arkworks-rs/curves/blob/1551d6d76ce5abf6e7925e53b0ea1af7dbc421c3/bw6_761/src/curves/mod.rs#L39), where `ATE_LOOP_COUNT_2` refers to the 2-NAF, but I see that apart from meaning 2-NAF it could mean the second loop count, which != `ATE_LOOP_COUNT`

Reply:
Resolved in [f56cce3](https://github.com/arkworks-rs/algebra/pull/445/commits/f56cce383781980f4cf8480383c22dc421698282).